### PR TITLE
W17 express Golden Globe API

### PR DESCRIPTION
**Pull request by Snabbina**
Endpoints in README.
https://golden-globes-technigo.herokuapp.com/

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -1,30 +1,43 @@
 import express from 'express'
 import bodyParser from 'body-parser'
 import cors from 'cors'
+import data from './data.json'
+
+console.log(data.length)
```
Comment (EmOlsz) [inline server.js:6]: Let's delete this console.log

File: server.js [hunk 2]
```diff
@@ -1,30 +1,43 @@
...
 // Add middlewares to enable cors and json body parsing
 app.use(cors())
+//to read json
 app.use(bodyParser.json())
 
-// Start defining your routes here
+//Start defining routes here
 app.get('/', (req, res) => {
-  res.send('Hello world')
+  res.send('Welcome to the Golden Globe API of nominations for 2010-2019')
```
Comment (EmOlsz) [inline server.js:20]: Would be better to send it as an object :

```
res.json({ message: 'Welcome to the Golden Globe API of nominations for 2010-2019' })
```

File: server.js [hunk 3]
```diff
@@ -1,30 +1,43 @@
...
+app.get('/nominations', (req, res) => {
+  res.json(data)
+
+})
+
+//Endpoint getting year and film won
+app.get('/years/:year', (req, res) => {
+  const year = req.params.year 
+  const showWon = req.query.won
+  console.log(showWon)
```
Comment (EmOlsz) [inline server.js:33]: Another console.log

**Review by EmOlsz — CHANGES_REQUESTED** [review]

**Comment by Snabbina** [comment]
Hey hey, @EmOlsz made some nice changes to the code, please let me know if it is ok now =) /Sabina

**Comment by EmOlsz** [comment]
Hey Sabina, thanks for fixes, project is now approved :)

**Review by EmOlsz — APPROVED** [review]

**Pull request closed without merging**